oprheum-game/SnowballNode.swift

```diff
@@ -22,6 +22,7 @@ class SnowballNode: SKNode {
     }
 
     func setSnowballMass(mass: CGFloat) {
+        if mass>1.6 {let mass = 1.8}
```

Comment:
Your code doesn't work because when you say `let mass = 1.8` you're creating a new variable `mass` that only exists between those two curly braces. You need to use a variable that exists outside of those curly braces.

I'm guessing you did it this way because this won't compile: 
![image](https://cloud.githubusercontent.com/assets/930039/19095740/30faf99c-8a5d-11e6-95cc-e3092c7a2845.png)

Try something more like:
```swift
var newMass = mass
if newMass > SnowballNode.maximumMass {
    newMass = Snowballnode.maximumMass
}
self.mass = newMass
```
Where `SnowballNode.maximumMass` is a constant defined above (like `SnowballNode.SnowballColor`)